ps_landing_page/models/hr_employee_landing_page.py

```diff
@@ -50,7 +50,7 @@ def _compute_all(self):
                           AND state != %s
                         GROUP BY employee_id ) hr2
                     on hr1.employee_id = hr2.employee_id""",
-                (self.employee_id.id, "validate", self.employee_id.id, "refuse"),
+                (self.employee_id.id, "validate", self.employee_id.id, "validate"),
```

Comment:
Hi @hbrunn @kumarinie : The reason I requested this change is my own lack of understanding of some of the technical workings of Odoo. I noticed that the landing page only showed a holiday balance when people had actually written holidays on top of their allocation. 

So, users thad only had allocated hours but no consumed hours showed 0. Since I wanted a quick solution I explain my take on the calculation to Sushma and she implemented it as such. 

If the _compute_total_allocation_used gives us the same result then I am fine to use it.

Reply:
you want to show there what the smartbutton on employees shows, right? That's `allocation_remaining_display`

Reply:
@hbrunn @Stephanvandenberg
I will be adding mentioned changes.